packages/postcss-themes/package.json

```
@@ -6,9 +6,10 @@
 	"license": "GPL-2.0-or-later",
 	"keywords": [
 		"wordpress",
+		"gutenberg",
 		"postcss",
 		"css",
-		"postcss-plugin",
+		"plugin",
```

```suggestion
		"plugin",
		"postcss-plugin",
```

apparently, [postcss-plugin](https://www.npmjs.com/search?q=keywords:postcss-plugin) is a thing